graphql-server/go-server/pkg/common/common.go

```diff
@@ -0,0 +1,95 @@
+package common
+
+import (
+	"context"
+	"fmt"
+
+	apichain "github.com/kubeagi/arcadia/api/app-node/chain/v1alpha1"
+	apiprompt "github.com/kubeagi/arcadia/api/app-node/prompt/v1alpha1"
+	arcadiav1alpha1 "github.com/kubeagi/arcadia/api/base/v1alpha1"
+	"github.com/kubeagi/arcadia/graphql-server/go-server/graph/generated"
+	corev1 "k8s.io/api/core/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+	"k8s.io/client-go/dynamic"
+	clientgoscheme "k8s.io/client-go/kubernetes/scheme"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+)
+
+var (
+	scheme   = runtime.NewScheme()
+	APIGroup = arcadiav1alpha1.GroupVersion.String()
+)
+
+func init() {
+	utilruntime.Must(clientgoscheme.AddToScheme(scheme))
+
+	utilruntime.Must(arcadiav1alpha1.AddToScheme(scheme))
+	utilruntime.Must(corev1.AddToScheme(scheme))
+	utilruntime.Must(apichain.AddToScheme(scheme))
+	utilruntime.Must(apiprompt.AddToScheme(scheme))
+}
+
+// MakeAuthSecretName returns a secret name based on its base name and owner's kind
+func MakeAuthSecretName(base string, ownerKind string) string {
```

Comment:
@Abirdcfly @0xff-dev we can move some common definitions,functions under this package